apps/api/src/app/app.module.ts

```diff
@@ -220,8 +220,17 @@ const autoSchemaFile = environment.production
       firearmLicense: {
         apiKey: environment.firearmLicense.pkPassApiKey!,
         apiUrl: environment.smartSolutionsApiUrl!,
-        passTemplateId: environment.firearmLicense
-          .firearmLicensePassTemplateId!,
+        passTemplateId: environment.firearmLicense.passTemplateId!,
+      },
+      machineLicense: {
+        apiKey: environment.machineLicense.pkPassApiKey!,
+        apiUrl: environment.smartSolutionsApiUrl!,
+        passTemplateId: environment.machineLicense.passTemplateId!,
+      },
+      adrLicense: {
+        apiKey: environment.adrLicense.pkPassApiKey!,
+        apiUrl: environment.smartSolutionsApiUrl!,
+        passTemplateId: environment.adrLicense.passTemplateId!,
```

Comment:
Please put on the schedule to change this module (and others you maintain) from the `Module.register(environment),` pattern to just `Module,` using ConfigModule instead.

We'd like to get rid of environment files completely.

Reply:
a refactor is next up